ReactSkia/components/RSkComponentScrollView.cpp

```diff
@@ -394,7 +398,10 @@ ScrollStatus RSkComponentScrollView::handleSnapToOffsetScroll(rnsKey direction,R
      if(nextOffset == currentOffset) return noScroll;
      if((contentLength - nextOffset) < frameLength)
         nextOffset = contentLength-frameLength;
+  } else if((direction == RNS_KEY_Left) || (direction == RNS_KEY_Up)) {
+     if(prevOffset == -1) return noScroll;
```

Comment:
Can we have a coding guideline for single line IF. It is always safe to use with braces

Reply:
okay agreed .. will update it